chevah_build

```diff
@@ -74,7 +84,11 @@ PYTHON_BIN=$INSTALL_FOLDER/bin/python
 PYTHON_BUILD_FOLDER="$PYTHON_VERSION-$OS-$ARCH"
 
 export MAKE=make
+
+# Used when building cffi on ArchLinux
 export CHEVAH_BUILD_PATH=$INSTALL_FOLDER
+# Used when building python on Windows or ArchLinux
+export PIP_ARGS
```

Comment:
Do we still need to export pip_args ? I was hopping that we reduce the scope of this var to this file